Checkout/Source/Network/Models/SecurityCode/SecurityCodeError.swift

```diff
@@ -0,0 +1,35 @@
+// 
+//  SecurityCodeError.swift
+//  
+//
+//  Created by Okhan Okbay on 05/10/2023.
+//
+
+import Foundation
+
+extension TokenisationError {
+  public enum SecurityCodeError: CheckoutError {
+    case cvvTokenInvalid
+    case cvvTokenUsed
+    case cvvTokenExpired
```

Comment:
which error codes shall we have here? @ehab-al-cko @deepesh-vasthimal-cko

Reply:
@okhan-okbay-cko there are no static error codes. We return directly what we are receiving from the tokenisation end point
You can test it in postman
curl --location 'https://api.sandbox.checkout.com/tokens' \
--header 'Content-Type: application/json' \
--header 'Authorization: pk_6b30805a-1f3b-4c63-8b75-eb3030109173' \
--header 'User-Agent: checkout-android/1.0.0' \
--data '{
  "type": "cvv",
  "token_data": {
    "cvv": ""
  }
}'

Reply:
It's an error that can be thrown from the `RequestFactory.swift` @deepesh-vasthimal-cko

Reply:
Are these error code just randomly defined by us for logging purposes? @ehab-al-cko @deepesh-vasthimal-cko This question is not relevant to the CVV component and overall a Frames question

Reply:
discussed this offline. on hold till next visit

Reply:
updated the error codes now. Following the same style of what iOS has been doing so far and returning an error code back to the merchants' developers. They'll need to assign the relevant error messages to these error codes as it's wanted to be done on Android.